adapters/pubmatic/pubmatic.go

```diff
@@ -48,6 +47,15 @@ type pubmaticParams struct {
 	Keywords    map[string]string `json:"keywords,omitempty"`
 }
 
+type pubmaticBidExtVideo struct {
+	Duration *int `json:"duration,omitempty"`
+}
+
+type pubmaticBidExt struct {
+	BidType           *int                 `json:"BidType,omitempty"`
```

Comment:
Is `BidType` the correct capitalization of this json field, or is this a copy/paste error?

Reply:
Yes, this field is coming like this only in the pubmatic response, we are going to change it in future.